assets/js/renderer.js

```
@@ -1,186 +1,179 @@
+  await window.utils.copyKey(data)
+
+  // Display tooltip for 5s
+  let label_id = `${lower_input_key}-key-text-area-tooltip`
+  document.getElementById(label_id).classList.add('text-green-500')
+  document.getElementById(label_id).innerText = `${input_key} Key copied`
+  setTimeout(function () {
+    remove_label_text(label_id)
+  }, 5000)
 }
 
 const saveKey = async (input_key) => {
```

ESLint

[error] Identifier 'input_key' is not in camel case.

(camelcase)